easybuild/easyconfigs/m/MXNet/MXNet-0.9.3-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MakeCp'
+
+name = 'MXNet'
+version = '0.9.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mxnet.io/' 
+description = """Flexible and Efficient Library for Deep Learning"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++11', 'opt': True, 'pic': True, 'openmp': True}
+
+#Download source from github, fix sub branches, checkout to version, 
+# clean the directory tree, make the tarball
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['MXNet-%(version)s_config.patch']
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('graphviz', '0.5.1', '-Python-%(pyver)s'),
+    ('OpenCV', '3.1.0'),
+]
+
+preconfigopts = ['cp make/config.mk . &&']
+
+buildopts = '&& mkdir -p %(builddir)s/mxnet/lib/python%(pyshortver)s/site-packages/ '
+buildopts += '&& cd python '
+buildopts += '&& env PYTHONPATH=$PYTHONPATH:%(builddir)s/mxnet/lib/python%(pyshortver)s/site-packages/ '
+buildopts += 'python setup.py install --prefix %(builddir)s/mxnet' 
+ 
+files_to_copy = ['lib']
+
+sanity_check_paths = {
+    'files': ['lib/libmxnet.%s' % SHLIB_EXT],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
```

Comment:
maybe also add a sanity check command to make sure importing the Python library works?

```python
sanity_check_commands = ["python -c 'import mxnet'"]